utility/src/main/java/org/oppia/android/util/logging/firebase/BUILD.bazel

```diff
@@ -1,77 +1,46 @@
 """
-General purpose utilities for firebase logging.
+Logging utility implementations backed by Firebase.
 """
 
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 
 kt_android_library(
-    name = "event_logger",
+    name = "impl",
     srcs = [
         "FirebaseEventLogger.kt",
+        "FirebaseExceptionLogger.kt",
+        "FirebaseLogUploader.kt",
     ],
-    visibility = ["//:oppia_api_visibility"],
+    visibility = ["//visibility:private"],
```

Comment:
You can actually just remove this--visibility is private by default.

Reply:
Done